src/Denner/Client/ServiceDescription/Articles.php

```diff
@@ -210,5 +213,38 @@
                 '$ref' => 'Sort',
             ),
         ),
+        'PricePropertiesParam' => array(
+            'description' => 'Price properties',
+            'location' => 'query',
+            'required' => false,
+            'type' => 'object',
+            'properties' => array(
+                'valid_on' => array(
+                    'description' => 'Price for day',
+                    'type' => 'string',
```

Comment:
Should add 

`'pattern': '^'20[0-9]{2}-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[0-1])$', // Year: 2000-2099, month: 01-12, day: 01-31`

?